openupgrade_scripts/scripts/account/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,497 @@
+---Models in module 'account'---
+obsolete model account.account.template
+obsolete model account.fiscal.position.account.template
+obsolete model account.fiscal.position.tax.template
+obsolete model account.fiscal.position.template
+obsolete model account.group.template
+obsolete model account.reconcile.model.line.template
+obsolete model account.reconcile.model.template
+obsolete model account.tax.repartition.line.template
+obsolete model account.tax.template
+# NOTHING TO DO: these models has been converted to CSV definition and existing company records XML-ID follow the same logic
```

Comment:
In pre-migration, I think we should call remove_tables_fks() for these tables, as hey have many2one to taxes.

also please include many2many tables there.